WordPress/src/main/java/org/wordpress/android/util/SiteUtils.java

```diff
@@ -87,4 +88,12 @@ public static boolean checkMinimalJetpackVersion(SiteModel site, String limitVer
         }
         return false;
     }
+
+    public static boolean isNonAtomicBusinessPlanSite(@Nullable SiteModel site) {
+        return site != null && !site.isAutomatedTransfer() && SiteUtils.hasBusinessPlan(site);
+    }
+
+    public static boolean hasBusinessPlan(SiteModel site) {
+        return site.getPlanShortName() != null && site.getPlanShortName().equalsIgnoreCase("business");
```

Comment:
Can we avoid hard-coding "business" here? Maybe this would work:

`return site.getPlanId() == PlansConstants.BUSINESS_PLAN_ID
               || site.getPlanId() == PlansConstants.JETPACK_BUSINESS_PLAN_ID;`

Reply:
Fixed in 11f057965f9382034b93566609a0c52cc7443f2a. I've decided to rename it and only return the non-jetpack business plans as the Jetpack ones should already have plugins feature available and it makes it easier to understand when Automated Transfer should be available. Let me know if you want me to change that back.